### ISSUE TITLE
Table: This table appears to be used for layout but does not have a role of presentation

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3848]
Description: This table appears to be used for layout but does not have a role of presentation

![image](https://user-images.githubusercontent.com/32538291/98776685-71f92a80-2415-11eb-8e15-4707682c1e5c.png)

Expected: Apply the role="presentation" to the table.

![image](https://user-images.githubusercontent.com/32538291/98776701-79b8cf00-2415-11eb-8cb4-5e1846fca6c2.png)

**Comment by IlyaSurmay** [comment]
This one is questionable because from what I know, `role="presentation"` should be applied in cases when `<table>` is used for layout purposes like email templates and other non-standard usages of  `<table>`, but in Table component `<table>` is used as a table - with rows, cells etc, adding `role="presentation"` will mislead screen reader software. 
I suggest closing this issue.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

As per the above reply,
The ticket can be closed.

Thanks and Regards
Sanjay
